### PR TITLE
Add examples of SYMMLQ and MINRES-QLP

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

**Comment by amontoison** [comment]
This pull request was forgotten.

File: examples/test_minres_qlp.jl [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+b = ones(n)
+b_norm = norm(b)
+
+# MINRES-QLP returns the minimum-norm solution of symmetric, singular and inconsistent systems
+(x, stats) = minres_qlp(A, b);
+r = b - A * x;
+
+@printf("Residual r: %s\n", Krylov.vec2str(r))
+@printf("Relative residual norm ‖r‖: %8.1e\n", norm(r) / b_norm)
+@printf("Solution x: %s\n", Krylov.vec2str(x))
```
Comment (dpo) [inline examples/test_minres_qlp.jl]: Let's check that x is indeed the minimum-norm solution.

File: examples/test_symmlq.jl [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+b = A * ones(n)
+b_norm = norm(b)
+
+# SYMMLQ returns the minimum-norm solution of symmetric, singular and consistent systems
+(x, stats) = symmlq(A, b, transfer_to_cg=false);
+r = b - A * x;
+
+@printf("Residual r: %s\n", Krylov.vec2str(r))
+@printf("Relative residual norm ‖r‖: %8.1e\n", norm(r) / b_norm)
+@printf("Solution x: %s\n", Krylov.vec2str(x))
```
Comment (dpo) [inline examples/test_symmlq.jl]: Here too.

File: examples/test_minres_qlp.jl (hunk 1 above)
Comment (amontoison) [reply examples/test_minres_qlp.jl]: How can I do that? A \ b returns with [1.0; 1.0; 1.0; Inf].

File: examples/test_symmlq.jl (hunk 2 above)
Comment (amontoison) [reply examples/test_symmlq.jl]: A \ b returns [1.0; 1.0; 1.0; NaN] with this problem.

File: examples/test_minres_qlp.jl (hunk 1 above)
Comment (dpo) [reply examples/test_minres_qlp.jl]: Your system isn't consistent. What did MINRES-QLP return here?

File: examples/test_symmlq.jl (hunk 2 above)
Comment (dpo) [reply examples/test_symmlq.jl]: Set `b = [1.0; 2.0; 3.0; 0.0]`. Then `x = [1.0; 1.0; 1.0; 0.0]`. By the way, CG should also identify the same solution.

File: examples/test_minres_qlp.jl (hunk 1 above)
Comment (amontoison) [reply examples/test_minres_qlp.jl]: MINRES-QLP returns [1.0; 1.0; 1.0; 0.0]. The minimum-norm solution that minimize ||b - Ax||.

Comment (dpo) [reply examples/test_minres_qlp.jl]: Yes, that seems right. Let's just check that that's the value returned.

**Comment by dpo** [comment]
Thank you.

**Pull request merged**